test/simplehashregistrar_test.js

```diff
@@ -122,7 +127,7 @@ describe('SimpleHashRegistrar', function() {
 			// Check a duplicate bid would throw
 			function(done) {
 				registrar.newBid.call(bid, {from: accounts[0], value: 2e18}, function(err, result) {
-					assert.ok(err.toString().indexOf(utils.INVALID_JUMP) != -1, err);
+					assert.ok(err, err);
```

Comment:
Yep, I think this snippet of the test is to ensure duplicate bid gets throw. So `err` is expected to be not `undefined` so `assert.ok(err)` would go through.